pkg/services/apiserver/standalone/factory.go

```diff
@@ -79,9 +79,9 @@ func (p *DummyAPIFactory) MakeAPIServer(gv schema.GroupVersion) (builder.APIGrou
 				Client: client.NewTestDataClient(),
 			},
 			client.NewTestDataRegistry(),
-			nil,                               // legacy lookup
-			prometheus.NewRegistry(),          // ???
-			tracing.InitializeTracerForTest(), // ???
```

Comment:
what I don't understand is why you need this change in this PR, which is just removing unused dependencies, since you are adding a new dependency. But we will need that eventually so feel free to go ahead.